mne/decoding/receptive_field.py

```diff
@@ -0,0 +1,353 @@
+import numpy as np
+from .base import _get_final_est, BaseEstimator, _check_estimator
+
+
+class ReceptiveField(BaseEstimator):
```

Comment:
Why wait until fitting? Both of these are determined during the `__init__` call. I don't see a need to have the trailing `_` since they aren't dependent on the fitting at all.